packages/manager/apps/web/client/app/domain/zone/activate/activate.controller.js

```diff
@@ -33,6 +33,8 @@ export default class DomainDnsZoneActivateController {
           [
             pricingConstants.PRICING_MODE.DEFAULTS.CREATE,
             pricingConstants.PRICING_MODE.DEFAULTS.DEFAULT,
+            pricingConstants.PRICING_MODE.DEFAULTS.TRANSFER,
+            pricingConstants.PRICING_MODE.DEFAULTS.PREMIUM,
```

Comment:
We shouldn't be maintaining those :/

Reply:
Yes sure, unfortunately API doesn't have any enumeration for them :(